tests/system_tests/tango/conftest.py

```diff
@@ -38,6 +38,14 @@ def pytest_collection_modifyitems(config, items):
                         reason="Tango is currently not supported on Python 3.12: https://github.com/bluesky/ophyd-async/issues/681"
                     )
                 )
+            if sys.platform.startswith(
+                "win"
+            ):  # expect "win32", but open to a future change: https://mail.python.org/pipermail/patches/2000-May/000648.html
+                item.add_marker(
+                    pytest.mark.skip(
+                        reason="Ophyd-async is currently not tested on Windows + Tango"
+                    )
+                )
```

Comment:
Can we use `@pytest.mark.skipif` instead? https://docs.pytest.org/en/stable/how-to/skipping.html#id1 Means that the skipping expectation is right next to the test and I believe removes the need for most of the `pytest_*` magics